tasks/eva_2323/dump_studies_json.py

```diff
@@ -24,7 +24,7 @@ def connect(self):
         conn = psycopg2.connect(pg_url, user=pg_user, password=pg_pass)
         return conn
 
-    def dump_from_database(self):
+    def dump_from_database(self, properties_file, stage):
```

Comment:
These two additional arguments don't seem to be used inside the function...

Reply:
That's true. I forgot to remove them.